Header header
float64[] elevation
float64[] map_center